pkg/solidity-utils/contracts/openzeppelin/Arrays.sol

```diff
@@ -0,0 +1,237 @@
+// SPDX-License-Identifier: MIT
+// OpenZeppelin Contracts (last updated v5.1.0) (utils/Arrays.sol)
+// This file was procedurally generated from scripts/generate/templates/Arrays.js.
+
+// NOTE: This file copied only the sort and necessary helper functions from Arrays.sol (OpenZeppelin Contracts v5.1.0)
```

Comment:
Are we copying it because it wasn't released in the version we're using? If so I'd just consider upgrading and deleting this file.